intbot/core/admin.py

```diff
@@ -67,5 +67,29 @@ def content_short(self, obj):
         return f"{obj.content[:10]}...{obj.content[-10:]}"
 
 
+class PretalxDataAdmin(admin.ModelAdmin):
+    list_display = [
+        "uuid",
+        "endpoint",
+        "created_at",
+        "modified_at",
+    ]
+    list_filter = ["created_at"]
```

Comment:
Why is this a filter?

Reply:
I copied the setup from the Webhook, but it's useful for debugging in admin to narrow down the list.